Looking in links: https://download.pytorch.org/whl/torch_stable.html
Requirement already satisfied: torch==1.5.0+cu101 in /home/jlpkort/anaconda3/lib/python3.8/site-packages (1.5.0+cu101)
Requirement already satisfied: torchvision==0.6.0+cu101 in /home/jlpkort/anaconda3/lib/python3.8/site-packages (0.6.0+cu101)
Requirement already satisfied: future in /home/jlpkort/anaconda3/lib/python3.8/site-packages (from torch==1.5.0+cu101) (0.18.2)
Requirement already satisfied: numpy in /home/jlpkort/anaconda3/lib/python3.8/site-packages (from torch==1.5.0+cu101) (1.17.4)
Requirement already satisfied: pillow>=4.1.1 in /home/jlpkort/anaconda3/lib/python3.8/site-packages (from torchvision==0.6.0+cu101) (8.2.0)
ERROR: Could not find a version that satisfies the requirement opencv_contrib_python_nonfree==4.1.1.1
ERROR: No matching distribution found for opencv_contrib_python_nonfree==4.1.1.1
wandb: Currently logged in as: tijmenk_ (use `wandb login --relogin` to force relogin)
DATASET:
  MASKS: ./datasets/masks_tv_256
  NAME: TVB
  ROOT: ./datasets/data_256
  SIZE: 256
  SIZE_H: 256
  SIZE_W: 256
MODEL:
  D:
    BETAS: (0.9, 0.99)
    DECAY_RATE: 0.5
    LR: 1e-05
    NAME: 1-ChOutputDiscriminator
    NUM_CHANNELS: 64
    NUM_CRITICS: 5
    SCHEDULER: []
  IS_TRAIN: True
  JOINT:
    BETAS: (0.9, 0.99)
    DECAY_RATE: 0.5
    LR: 0.0002
    NAME: JointNetwork
    SCHEDULER: []
  MPN:
    BETAS: (0.9, 0.99)
    DECAY_RATE: 0.0
    LOSS_COEFF: 2.0
    LR: 1e-05
    NAME: MaskPredictionNetwork
    NECK_CHANNELS: 128
    NUM_CHANNELS: 64
    SCHEDULER: []
  NAME: VCNet
  RIN:
    BETAS: (0.5, 0.9)
    DECAY_RATE: 0.0
    EMBRACE: True
    LOSS_COEFF: 1.0
    LR: 0.0001
    NAME: RobustInpaintingNetwork
    NECK_CHANNELS: 128
    NUM_CHANNELS: 32
    SCHEDULER: []
OPTIM:
  ADVERSARIAL: 0.001
  GP: 10
  MASK: 1
  RECON: 1.4
  SEMANTIC: 0.0001
  TEXTURE: 0.001
SYSTEM:
  NUM_GPU: 4
  NUM_WORKERS: 16
TEST:
  ABLATION: False
  BATCH_SIZE: 64
  C_IMG_ID: 38
  IMG_ID: 52
  ITER: 4
  MODE: 7
  OUTPUT_DIR: ./outputs
  WEIGHTS: None
TRAIN:
  BATCH_SIZE: 4
  LOG_INTERVAL: 1000
  NUM_STEPS_FOR_JOINT: 6600
  NUM_TOTAL_STEP: 100000
  RESUME: False
  SAVE_DIR: ./weights
  SAVE_INTERVAL: 10000
  SHUFFLE: True
  START_STEP: 0
  TUNE: False
  VISUALIZE_INTERVAL: 1000
WANDB:
  ENTITY: tijmenk_
  LOG_DIR: 
  NUM_ROW: 0
  PROJECT_NAME: thesis_pytorch
  RUN: 16
wandb: wandb version 0.12.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.11.0
wandb: Syncing run rosy-lake-125
wandb:  View project at https://wandb.ai/tijmenk_/thesis_pytorch
wandb:  View run at https://wandb.ai/tijmenk_/thesis_pytorch/runs/yx1j4svr
wandb: Run data is saved locally in /home/jlpkort/vcnet-blind-image-inpainting/wandb/run-20210907_122625-yx1j4svr
wandb: Run `wandb offline` to turn off syncing.
I0907 12:26:33.493886 19453 trainer.py:244] Using 4 GPUs...

CHANNELS:  64
CHANNELS:  64
Traceback (most recent call last):
  File "/home/jlpkort/anaconda3/lib/python3.8/multiprocessing/util.py", line 300, in _run_finalizers
    finalizer()
  File "/home/jlpkort/anaconda3/lib/python3.8/multiprocessing/util.py", line 224, in __call__
    res = self._callback(*self._args, **self._kwargs)
  File "/home/jlpkort/anaconda3/lib/python3.8/multiprocessing/util.py", line 133, in _remove_temp_dir
    rmtree(tempdir)
  File "/home/jlpkort/anaconda3/lib/python3.8/shutil.py", line 722, in rmtree
    onerror(os.rmdir, path, sys.exc_info())
  File "/home/jlpkort/anaconda3/lib/python3.8/shutil.py", line 720, in rmtree
    os.rmdir(path)
OSError: [Errno 39] Directory not empty: '/scratch/pymp-pbu16na5'
slurmstepd: error: *** JOB 8168045 ON r30n2 CANCELLED AT 2021-09-07T13:26:19 DUE TO TIME LIMIT ***
